### DISCUSSION TITLE
Errors exporting raw pcap in SO2

### DISCUSSION THREAD

**Discussion by cyb3rslack3r** [Security-Onion-Solutions/securityonion#4155] [Unsupported Versions]
Although my other question about exporting readable pcap from stenographer was unanswered, I found some ways to export pcap, but have run across an issue where the pcap stops exporting after 70 to 80GBs.  I'm not sure why it seems to error out during this level.  

Here is the command I am using: docker exec -it so-sensoroni scripts/stenoquery.sh 'net 0.0.0.0/0' -w /nsm/pcapout/test.pcap 

I am using this command now to try and chop it up, but would prefer a more automated way to export pcap from stenographer:  docker exec -it so-sensoroni scripts/stenoquery.sh "after 2020-06-09T00:00:00Z and before 2020-06-10T00:00:00Z" -w /nsm/pcapout/some.pcap

Any help would be appreciated

**Comment by TOoSmOotH** [comment]
I would probably split it by hours or minutes depending on your traffic

**Reply by cyb3rslack3r** [reply]
That is the current process.  We are trying to create a script to automate the process since it requires too much user interaction with TBs of data.

**Comment by defensivedepth** [comment]
@cyb3rslack3r 

You are probably hitting either a timeout or max pcap bytes - they can be overridden by:

`STENO_TIMEOUT=3600 STENO_MAX_PCAP_BYTES=xyz stenoquery.sh <query>`

**Reply by cyb3rslack3r** [reply]
Just got around to testing this.  I initially adjusted the max pcap bytes setting when I ran across the 2gb file write limit, but not the timeout setting.  After modifying the "STENO_TIMEOUT, I don't get an error where it stops the process, but it is still not copying all the pcap over in the command.  If you didn't know the original size you would probably not know any better, but the combined files are over 200 gb and the output file stops with no errors at 68gb.
